lib/public/views/Runs/format/formatRunDetectorsInline.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * @license
+ * Copyright CERN and copyright holders of ALICE O2. This software is
+ * distributed under the terms of the GNU General Public License v3 (GPL
+ * Version 3), copied verbatim in the file "COPYING".
+ *
+ * See http://alice-o2.web.cern.ch/license for full licensing information.
+ *
+ * In applying this license CERN does not waive the privileges and immunities
+ * granted to it by virtue of its status as an Intergovernmental Organization
+ * or submit itself to any jurisdiction.
+ */
+
+import { h } from '/js/src/index.js';
+import { RunDetectorQualities } from '../../../domain/enums/RunDetectorQualities.js';
+
+/**
+ * Format a list of detectors inline
+ *
+ * @return {Component} the formatted run detectors inline
+ * @param {RunDetectorQuality[]} detectorsQualities The run detector quality per detector
+ * @param {number} nDetectors The number of run detectors
+ */
+export const formatRunDetectorsInline = (detectorsQualities, nDetectors) => {
+    if (detectorsQualities && detectorsQualities.length > 0) {
```

Comment:
```suggestion
    if (detectorsQualities?.length > 0) {
```
Browser supported by our applications are all available to use the optional chaining. [More](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining)